.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,62 @@
+---
+name: Bug report
+about: Create a report to help us improve
```

Comment:
What about:

Did you encounter something unexpected or incorrect in the Dataverse software? We'd like to hear about it!